pypesto/ensemble/ensemble.py

```diff
@@ -1220,6 +1224,10 @@ def calculate_cutoff(result: Result, percentile: float = 0.95):
         The percentile of the chi^2 distribution. Between 0 and 100.
         Higher values will result in a more lax cutoff. If the value is greater
         than 100, the cutoff will be returned as np.inf.
+    cr_option:
+        The type of confidence region, which determines the degree of freedom of
+        the chi^2 distribution for the cutoff value. It can take 'simultaneous' or
+        'pointwise'.
```

Comment:
A small link to a paper to what the difference is would be nice for users.

Reply:
Agreed, but some internet article, e.g. Wikipedia, would also be fine for me